/var/lib/condor/execute/slot1/dir_71032/condor_exec.exe: line 8: lspci: command not found
  0%|          | 0/36 [00:00<?, ?it/s]Extracting : pysocks-1.7.1-py37_0.conda:   0%|          | 0/36 [00:00<?, ?it/s]Extracting : pysocks-1.7.1-py37_0.conda:   3%|▎         | 1/36 [00:00<00:04,  7.93it/s]Extracting : tqdm-4.42.1-py_0.conda:   3%|▎         | 1/36 [00:00<00:04,  7.93it/s]    Extracting : zlib-1.2.11-h7b6447c_3.conda:   6%|▌         | 2/36 [00:00<00:04,  7.93it/s]Extracting : zlib-1.2.11-h7b6447c_3.conda:   8%|▊         | 3/36 [00:00<00:04,  7.71it/s]Extracting : libgcc-ng-9.1.0-hdf63c60_0.conda:   8%|▊         | 3/36 [00:00<00:04,  7.71it/s]Extracting : libgcc-ng-9.1.0-hdf63c60_0.conda:  11%|█         | 4/36 [00:00<00:03,  8.12it/s]Extracting : ld_impl_linux-64-2.33.1-h53a641e_7.conda:  11%|█         | 4/36 [00:00<00:03,  8.12it/s]Extracting : yaml-0.1.7-had09818_2.conda:  14%|█▍        | 5/36 [00:00<00:03,  8.12it/s]             Extracting : tk-8.6.8-hbc83047_0.conda:  17%|█▋        | 6/36 [00:00<00:03,  8.12it/s]  Extracting : setuptools-45.2.0-py37_0.conda:  19%|█▉        | 7/36 [00:00<00:03,  8.12it/s]Extracting : chardet-3.0.4-py37_1003.conda:  22%|██▏       | 8/36 [00:00<00:03,  8.12it/s] Extracting : libstdcxx-ng-9.1.0-hdf63c60_0.conda:  25%|██▌       | 9/36 [00:00<00:03,  8.12it/s]Extracting : requests-2.22.0-py37_1.conda:  28%|██▊       | 10/36 [00:00<00:03,  8.12it/s]      Extracting : conda-package-handling-1.6.0-py37h7b6447c_0.conda:  31%|███       | 11/36 [00:00<00:03,  8.12it/s]Extracting : idna-2.8-py37_0.conda:  33%|███▎      | 12/36 [00:00<00:02,  8.12it/s]                            Extracting : wheel-0.34.2-py37_0.conda:  36%|███▌      | 13/36 [00:00<00:02,  8.12it/s]Extracting : python-3.7.6-h0371630_2.conda:  39%|███▉      | 14/36 [00:01<00:02,  8.12it/s]Extracting : python-3.7.6-h0371630_2.conda:  42%|████▏     | 15/36 [00:01<00:02,  9.66it/s]Extracting : libffi-3.2.1-hd88cf55_4.conda:  42%|████▏     | 15/36 [00:01<00:02,  9.66it/s]Extracting : cffi-1.14.0-py37h2e261b9_0.conda:  44%|████▍     | 16/36 [00:01<00:02,  9.66it/s]Extracting : pip-20.0.2-py37_1.conda:  47%|████▋     | 17/36 [00:01<00:01,  9.66it/s]         Extracting : libedit-3.1.20181209-hc058e9b_0.conda:  50%|█████     | 18/36 [00:01<00:01,  9.66it/s]Extracting : ca-certificates-2020.1.1-0.conda:  53%|█████▎    | 19/36 [00:01<00:01,  9.66it/s]     Extracting : openssl-1.1.1d-h7b6447c_4.conda:  56%|█████▌    | 20/36 [00:01<00:01,  9.66it/s] Extracting : cryptography-2.8-py37h1ba5d50_0.conda:  58%|█████▊    | 21/36 [00:01<00:01,  9.66it/s]Extracting : six-1.14.0-py37_0.conda:  61%|██████    | 22/36 [00:01<00:01,  9.66it/s]              Extracting : certifi-2019.11.28-py37_0.conda:  64%|██████▍   | 23/36 [00:01<00:01,  9.66it/s]Extracting : asn1crypto-1.3.0-py37_0.conda:  67%|██████▋   | 24/36 [00:01<00:01,  9.66it/s]  Extracting : pyopenssl-19.1.0-py37_0.conda:  69%|██████▉   | 25/36 [00:01<00:01,  9.66it/s]Extracting : sqlite-3.31.1-h7b6447c_0.conda:  72%|███████▏  | 26/36 [00:01<00:01,  9.66it/s]Extracting : ruamel_yaml-0.15.87-py37h7b6447c_0.conda:  75%|███████▌  | 27/36 [00:01<00:00,  9.66it/s]Extracting : _libgcc_mutex-0.1-main.conda:  78%|███████▊  | 28/36 [00:01<00:00,  9.66it/s]            Extracting : urllib3-1.25.8-py37_0.conda:  81%|████████  | 29/36 [00:01<00:00,  9.66it/s] Extracting : pycparser-2.19-py37_0.conda:  83%|████████▎ | 30/36 [00:01<00:00,  9.66it/s]Extracting : xz-5.2.4-h14c3975_4.conda:  86%|████████▌ | 31/36 [00:01<00:00,  9.66it/s]  Extracting : pycosat-0.6.3-py37h7b6447c_0.conda:  89%|████████▉ | 32/36 [00:01<00:00,  9.66it/s]Extracting : readline-7.0-h7b6447c_5.conda:  92%|█████████▏| 33/36 [00:01<00:00,  9.66it/s]     Extracting : ncurses-6.2-he6710b0_0.tar.bz2:  94%|█████████▍| 34/36 [00:01<00:00,  9.66it/s]Extracting : conda-4.8.2-py37_0.tar.bz2:  97%|█████████▋| 35/36 [00:01<00:00,  9.66it/s]                                                                                            Building graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]Examining @/linux-64::__glibc==2.17=0:   0%|          | 0/6 [00:00<?, ?it/s]Examining python=3.7:  17%|█▋        | 1/6 [00:00<00:00, 10180.35it/s]      Examining pytorch::torchvision=0.6:  33%|███▎      | 2/6 [00:00<00:00, 13.33it/s]Examining pytorch::torchvision=0.6:  50%|█████     | 3/6 [00:00<00:00, 19.98it/s]Examining cudatoolkit=10.2:  50%|█████     | 3/6 [00:02<00:00, 19.98it/s]        Examining cudatoolkit=10.2:  67%|██████▋   | 4/6 [00:02<00:01,  1.63it/s]Examining pytorch::pytorch=1.8:  67%|██████▋   | 4/6 [00:02<00:01,  1.63it/s]Examining @/linux-64::__cuda==11.5=0:  83%|████████▎ | 5/6 [00:02<00:00,  1.63it/s]Examining @/linux-64::__cuda==11.5=0: 100%|██████████| 6/6 [00:02<00:00,  2.07it/s]                                                                                   Determining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for cudatoolkit __glibc python:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for torchvision python pytorch:  17%|█▋        | 1/6 [00:00<00:00, 54.95it/s]Examining conflict for cudatoolkit torchvision python pytorch:  33%|███▎      | 2/6 [00:00<00:00,  4.12it/s]Examining conflict for cudatoolkit torchvision python pytorch:  50%|█████     | 3/6 [00:00<00:00,  6.18it/s]Examining conflict for cudatoolkit python:  50%|█████     | 3/6 [00:00<00:00,  6.18it/s]                    Examining conflict for cudatoolkit python:  67%|██████▋   | 4/6 [00:00<00:00,  3.86it/s]Examining conflict for torchvision pytorch:  67%|██████▋   | 4/6 [00:00<00:00,  3.86it/s]Examining conflict for cudatoolkit torchvision pytorch:  83%|████████▎ | 5/6 [00:01<00:00,  3.86it/s]Examining conflict for cudatoolkit torchvision pytorch: 100%|██████████| 6/6 [00:01<00:00,  4.06it/s]Examining conflict for __cuda: 100%|██████████| 6/6 [00:01<00:00,  4.06it/s]                         Examining conflict for __cuda: : 7it [00:01,  3.19it/s]                                                                            Building graph of deps:   0%|          | 0/6 [00:00<?, ?it/s]Examining @/linux-64::__glibc==2.17=0:   0%|          | 0/6 [00:00<?, ?it/s]Examining python=3.7:  17%|█▋        | 1/6 [00:00<00:00, 9341.43it/s]       Examining pytorch::torchvision=0.6:  33%|███▎      | 2/6 [00:00<00:00, 12.52it/s]Examining pytorch::torchvision=0.6:  50%|█████     | 3/6 [00:00<00:00, 18.77it/s]Examining cudatoolkit=10.2:  50%|█████     | 3/6 [00:02<00:00, 18.77it/s]        Examining cudatoolkit=10.2:  67%|██████▋   | 4/6 [00:02<00:01,  1.57it/s]Examining pytorch::pytorch=1.8:  67%|██████▋   | 4/6 [00:02<00:01,  1.57it/s]Examining @/linux-64::__cuda==11.5=0:  83%|████████▎ | 5/6 [00:02<00:00,  1.57it/s]Examining @/linux-64::__cuda==11.5=0: 100%|██████████| 6/6 [00:02<00:00,  1.99it/s]                                                                                   Determining conflicts:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for cudatoolkit __glibc python:   0%|          | 0/6 [00:00<?, ?it/s]Examining conflict for torchvision python pytorch:  17%|█▋        | 1/6 [00:00<00:00, 52.27it/s]Examining conflict for cudatoolkit torchvision python pytorch:  33%|███▎      | 2/6 [00:00<00:01,  3.91it/s]Examining conflict for cudatoolkit torchvision python pytorch:  50%|█████     | 3/6 [00:00<00:00,  5.86it/s]Examining conflict for cudatoolkit python:  50%|█████     | 3/6 [00:01<00:00,  5.86it/s]                    Examining conflict for cudatoolkit python:  67%|██████▋   | 4/6 [00:01<00:00,  3.67it/s]Examining conflict for torchvision pytorch:  67%|██████▋   | 4/6 [00:01<00:00,  3.67it/s]Examining conflict for cudatoolkit torchvision pytorch:  83%|████████▎ | 5/6 [00:01<00:00,  3.67it/s]Examining conflict for cudatoolkit torchvision pytorch: 100%|██████████| 6/6 [00:01<00:00,  3.88it/s]Examining conflict for __cuda: 100%|██████████| 6/6 [00:01<00:00,  3.88it/s]                         Examining conflict for __cuda: : 7it [00:01,  3.05it/s]                                                                            
UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package _libgcc_mutex conflicts for:
cudatoolkit=10.2 -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build=main]
python=3.7 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1',build=main]

Package __glibc conflicts for:
cudatoolkit=10.2 -> libgcc-ng[version='>=7.3.0'] -> __glibc[version='>=2.17']
@/linux-64::__glibc==2.17=0
python=3.7 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']

Package cudatoolkit conflicts for:
cudatoolkit=10.2
pytorch::pytorch=1.8 -> cudatoolkit[version='>=10.1,<10.2|>=11.1,<11.2|>=10.2,<10.3']
pytorch::torchvision=0.6 -> cudatoolkit[version='>=10.1,<10.2|>=10.2,<10.3|>=9.2,<9.3']

Package python conflicts for:
pytorch::pytorch=1.8 -> ninja -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.5,<3.6.0a0|>=3.6|>=3.5|>=3.7|>=3.6,<3.7']
python=3.7
pytorch::pytorch=1.8 -> python[version='>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0|>=3.9,<3.10.0a0']
pytorch::torchvision=0.6 -> python[version='>=3.5,<3.6.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']
pytorch::torchvision=0.6 -> numpy[version='>=1.11'] -> python[version='>=2.7,<2.8.0a0|>=3.10,<3.11.0a0|>=3.9,<3.10.0a0']

Package libstdcxx-ng conflicts for:
pytorch::pytorch=1.8 -> ninja -> libstdcxx-ng[version='>=7.2.0|>=7.3.0|>=7.5.0']
cudatoolkit=10.2 -> libstdcxx-ng[version='>=7.3.0']
pytorch::torchvision=0.6 -> cudatoolkit[version='>=10.2,<10.3'] -> libstdcxx-ng[version='>=7.2.0|>=7.3.0|>=7.5.0|>=8.4.0']
python=3.7 -> libstdcxx-ng[version='>=7.2.0|>=7.3.0']

Package pytorch conflicts for:
pytorch::torchvision=0.6 -> pytorch[version='1.5.0|1.5.1']
pytorch|pytorch::pytorch=1.8
pytorch::pytorch=1.8The following specifications were found to be incompatible with your CUDA driver:

  - feature:/linux-64::__cuda==11.5=0
  - feature:|@/linux-64::__cuda==11.5=0

Your installed CUDA driver is: 11.5


Could not find conda environment: pytorch-gpu
You can list all discoverable environments with `conda info --envs`.

Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'torch'
/var/lib/condor/execute/slot1/dir_71032/condor_exec.exe: line 45: start: command not found
Traceback (most recent call last):
  File "main.py", line 40, in <module>
    import torch
ModuleNotFoundError: No module named 'torch'
/var/lib/condor/execute/slot1/dir_71032/condor_exec.exe: line 50: date +%s: syntax error: operand expected (error token is "%s")
/var/lib/condor/execute/slot1/dir_71032/condor_exec.exe: line 51: runtime: command not found
